evutil.c

```diff
@@ -405,10 +405,12 @@ evutil_win_socketpair(int family, int type, int protocol,
 int
 evutil_socketpair(int family, int type, int protocol, evutil_socket_t fd[2])
 {
-#ifndef _WIN32
+#if defined(_WIN32)
+	return evutil_win_socketpair(family, type, protocol, fd);
+#elif defined(EVENT__HAVE_SOCKETPAIR)
 	return socketpair(family, type, protocol, fd);
 #else
-	return evutil_win_socketpair(family, type, protocol, fd);
+	evutil_ersatz_socketpair_(family, type, protocol, fd);
```

Comment:
```suggestion
	return evutil_ersatz_socketpair_(family, type, protocol, fd);
```

Reply:
Apparently you are not allowed edits for the maintainers in your repository, please apply this suggestion.

Reply:
Added the return, sorry about the mistake. Good that you caught it.